src/victory-util/events.js

```diff
@@ -125,7 +125,7 @@ export default {
         const mutationKeys = getKeys(childName);
         return Array.isArray(mutationKeys) ?
           mutationKeys.reduce((memo, key) => {
-            return assign(memo, getMutationObject(key, childName));
+            return assign({}, memo, getMutationObject(key, childName));
```

Comment:
Why aren't we mutating `memo`? Seems like extra work to copy all the props for no benefit (?)